fabric-content-registries-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,27 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-content-registries-v1",
+  "name": "Fabric Content Registries (v1)",
+  "version": "${version}",
+  "environment": "*",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-content-registries-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
+    "fabric-api-base": "*",
+    "fabric-resource-loader-v0": "*"
+  },
+  "description": "Adds registries for vanilla mechanics that are missing them.",
+  "mixins": [
+    "fabric-content-registries-v1.mixins.json"
+  ]
```

Comment:
This module now needs to define a module lifecycle:
For the new module you would add the following to the fabric.mod.json
```json
  "custom": {
    "fabric-api:module-lifecycle": "stable"
  }
```